app/services/utilities/employment_income_monthly_equivalent_calculator.rb

```diff
@@ -49,22 +49,26 @@ def weekly_to_monthly(value)
       (value * 52 / 12).round(2)
     end
 
-    def blunt_average(_value)
-      @employment.employment_payments.sum(&:gross_income) / 3
+    def blunt_average(value)
+      (@employment.employment_payments.sum(&value) / @employment.employment_payments.count).round(2)
```

Comment:
this is the `attribute` being passed in i believe
```suggestion
    def blunt_average(attribute)
      (@employment.employment_payments.sum(&attribute) / @employment.employment_payments.count).round(2)
```